Common/GPU/OpenGL/GLQueueRunner.cpp

```diff
@@ -711,7 +711,7 @@ void GLQueueRunner::RunSteps(const std::vector<GLRStep *> &steps, bool skipGLCal
 		case GLRStepType::RENDER:
 			renderCount++;
 			if (IsVREnabled()) {
-				GLRStep vrStep = step;
+				GLRStep &vrStep = step;
```

Comment:
Hm, why not just use `step` then?  I assume this is because it was const before.

-[Unknown]